src/DGtal/helpers/ShortcutsGeometry.h

```diff
@@ -976,6 +981,132 @@ namespace DGtal
           return mc_estimations;
         }
 
+      
+        /// Given a digital shape \a bimage, a sequence of \a surfels,
```

Comment:
Please remove two spaces of indentation to be aligned with previous functions

Reply:
Thx